static/js/submit-warning.js

```diff
@@ -0,0 +1,53 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Since jQuery is already being loaded for bootstrap, it would probably make sense to make use of it here too.

Using jQuery's [`.ready()`](https://api.jquery.com/ready/) is more robust than `DOMContentLoaded` (since it fires even if things are loaded in the wrong order).

Reply:
Great suggestion, I've rewritten with jQuery.